src/test/bash/create_certificates.sh

```diff
@@ -67,7 +67,7 @@ chmod 400 ${CA_DIR}/private/localhost.decrypted.key.pem
 
 echo "[INFO] Generating server certificate request"
 openssl req -config <(cat ${DIR}/openssl.cnf \
-        <(printf "\n[SAN]\nsubjectAltName=DNS:localhost,IP:127.0.0.1")) \
+        <(printf "\n[SAN]\nsubjectAltName=DNS:localhost,IP:127.0.0.1,DNS:chanchan2")) \
```

Comment:
Care to elaborate on why this change is?

Reply:
Sorry, another local testing change I should have stashed and not checked in.